.github/workflows/format.yml

```diff
@@ -13,8 +13,23 @@ jobs:
     - uses: haskell-actions/run-fourmolu@v11
       with:
         version: "0.12.0.0"
+        # NOTE: Keep this in sync with `../../Makefile`
         pattern: |
+          bootstrap/**/*.hs
+          buildinfo-reference-generator/**/*.hs
           Cabal/**/*.hs
-          Cabal-syntax/**/*.hs
+          Cabal-benchmarks/**/*.hs
+          Cabal-described/**/*.hs
+          Cabal-dev-scripts/**/*.hs
+          Cabal-hooks/**/*.hs
           cabal-install/**/*.hs
+          cabal-install-solver/**/*.hs
+          Cabal-QuickCheck/**/*.hs
+          Cabal-syntax/**/*.hs
+          Cabal-tests/**/*.hs
+          Cabal-testsuite/src/**/*.hs
+          Cabal-testsuite/main/**/*.hs
+          Cabal-testsuite/static/**/*.hs
```

Comment:
Case seems off here (and maybe somewhere else)? I mean `Cabal-testsuite` not `cabal-testsuite`, and the latter is the correct one. 

It's also strange that CI missed this issue.